stories/intermission.stories.jsx

```diff
@@ -0,0 +1,99 @@
+// https://github.com/EnCiv/civil-pursuit/issues/137
+
+import React from 'react'
+import Intermission from '../app/components/intermission'
+import { INITIAL_VIEWPORTS } from '@storybook/addon-viewport'
+import {onDoneDecorator, onDoneResult } from './common'
+
+export default {
+  component: Intermission,
+  decorators: [onDoneDecorator],
+  parameters: {
+    viewport: {
+      viewports: INITIAL_VIEWPORTS,
+    },
+  },
+}
+
+const validateEmail = email => {
+  var re =
+    /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/
+  return re.test(String(email).toLowerCase())
+}
+
+// const Template = args => <Intermission {...args} />
+const Template = args => {
+  window.socket = {
+    emit: (handle, data, cb) => {
+      console.log('User input email:', data.email)
+      if (handle !== 'set user info') {
+        console.error('emit expected set user info, got:', handle)
+      }
+      if (validateEmail(data.email)) {
+        setTimeout(() => cb({ error: '' }), 1000)
+        console.log("success!")
+      } else {
+        setTimeout(() => cb({ error: 'Something went wrong' }), 1000)
+        console.log("error")
+      }
+    },
+  }
+
+  console.log('window.socket.emit defined inside Template')
+  return <Intermission {...args} />
+}
+
+export const Empty = Template.bind({})
+Empty.args = {}
+
+export const NoEmail = Template.bind({})
+NoEmail.args = {
+  user: {},
+}
+
+export const NoEmailMobile = Template.bind({})
+NoEmailMobile.args = {
+  user: {},
+}
+NoEmailMobile.parameters = {
+  viewport: {
+    defaultViewport: 'iphonex',
+  },
+}
+export const LastRound1 = Template.bind({})
+LastRound1.args = {
+  user: { email: 'example@gmail.com', tempid: '123456' },
+}
+
+
+export const LastRound2 = Template.bind({})
+LastRound2.args = {
+  user: { email: 'example@gmail.com', tempid: '123456' },
+  lastRound: 2,
+}
+LastRound2.play = async ({ canvasElement }) => {
+  const canvas = within(canvasElement)
+
+  const continueButton = await canvas.getByText('Yes, Continue')
+  await userEvent.click(continueButton)
```

Comment:
For all these stories, please have round, lastRound, and user all args to the story.  That way people exercise the different arg combinations.

For the LastRound2 story, Storybook's UI is weak in this area, but if you click on the Interactions tab you will see that it is failing the "play" tests.  Looks like just an import problem, but then you may need to make tweeks to make them pass. 

![image](https://github.com/user-attachments/assets/d52ed26e-9783-4012-a3f6-5b266c7833c8)